### DISCUSSION TITLE
Which output format is required?

### DISCUSSION THREAD

**Discussion by postb99** [McShelby/hugo-theme-relearn#704] [Q&A] [answered]
Hello,

I noticed that published website contains an `index.xml` file besides the `index.html`. What is the XML file for?

I don't need RSS, only HTML pages and search, can I update this configuration? Just remove `"RSS"`? I guess this will have a great impact on site building.

Maybe I need to read more documentation but it's too late tonight :)

```
[outputs]
home = ["HTML", "RSS", "SEARCH", "SEARCHPAGE"]
section = ["HTML", "RSS"]
page = ["HTML", "RSS", "PRINT"]
```

**Comment by McShelby** [comment, marked as answer]
Yeah, just remove the RSS and no `.xml` will be generated.
